app/src/main/java/com/example/myapplication/MatchData.java

```diff
@@ -59,28 +59,46 @@ public String returnAllData() {
     //Getters and Setters
```

Comment:
Why is this here?

Reply:
it creates a basic description of what this part of the file is about